libpod/image/search.go

```diff
@@ -147,6 +153,45 @@ func searchImageInRegistry(term string, registry string, options SearchOptions)
 	// every types.SystemContext, and to compute the value just once in one
 	// place.
 	sc.SystemRegistriesConfPath = sysreg.SystemRegistriesConfPath()
+	if options.ListTags {
```

Comment:
Can we break this out into a new function?

Reply:
@QiWang19 this was the @vrothberg breakout.  So the change he was asking for was what you did with `searchRepositoryTags()` function below?  If so, I wonder why GitHub didn't not that.  Maybe because this line didn't change?